packages/settings-library/src/settings_library/postgres.py

```diff
@@ -53,6 +53,12 @@ def dsn(self) -> str:
             port=f"{self.POSTGRES_PORT}",
             path=f"/{self.POSTGRES_DB}",
         )
+        return dsn
+
+    @cached_property
+    def dsn_with_query(self) -> str:
+        """Some clients do not support queries in the dsn"""
```

Comment:
really? which ones?

Reply:
See e.g. this  [code](https://github.com/ITISFoundation/osparc-simcore/blob/f91f7ea63f20a9549db6352b25e90ce2099028fc/packages/service-library/src/servicelib/common_aiopg_utils.py#L48) with ``aiopg.sa.create_engine``. the application name cannot be passed as a part of the dsn